ambry-clustermap/src/main/java/com/github/ambry/clustermap/HelixClusterManager.java

```diff
@@ -653,6 +654,15 @@ private void addClusterWideRawCapacity(long diskRawCapacityBytes) {
     clusterWideRawCapacityBytes.getAndAdd(diskRawCapacityBytes);
   }
 
+  /**
+   * Reduce cluster-wide raw capacity. This is called when new disk is removed from cluster. We update
+   * cluster-wide raw capacity by removing the disk capacity.
+   * @param diskRawCapacityBytes raw disk capacity to be decreased.
+   */
+  private void reduceClusterWideRawCapacity(long diskRawCapacityBytes) {
+    clusterWideRawCapacityBytes.getAndAdd(-diskRawCapacityBytes);
```

Comment:
single usage one liner. Replace its call site with this one liner itself.

Reply:
Done